pkg/cmd/server/start/start_master.go

```diff
@@ -330,6 +330,7 @@ func StartMaster(openshiftMasterConfig *configapi.MasterConfig) error {
 	}()
 
 	// Must start policy caching immediately
+	openshiftConfig.RunGroupCache()
```

Comment:
you're right... it was the `NewREST` bit in the cache construction that threw me